packages/api/lib/writeRecords/write-granules.js

```diff
@@ -979,6 +986,7 @@ const updateGranuleStatusToQueued = async (params) => {
       knex,
       snsEventType: 'Update',
       esClient,
+      execution: undefined,
```

Comment:
This function doesn't take an object with an execution. What is this doing?

Reply:
That was vestigial from another update I opted to leave out of this PR.  Good catch.  4269a6a59c